include/alpaka/vec/Vec.hpp

```diff
@@ -402,6 +402,34 @@ namespace alpaka
             return r;
         }
 
+        //! \return The element-wise minimum of one or more vectors.
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename... Vecs, typename = std::enable_if_t<(std::is_same_v<Vec, Vecs> && ...)>>
+        ALPAKA_FN_HOST_ACC friend constexpr auto elementwise_min(Vec const& p, Vecs const&... qs) -> Vec
```

Comment:
```suggestion
        template<typename... Vecs, typename = std::enable_if_t<(std::is_same_v<Vec, Vecs> && ... && true)>>
        ALPAKA_FN_HOST_ACC friend constexpr auto elementwise_min(Vec const& p, Vecs const&... qs) -> Vec
```
To support `elementwise_min` with a single argument.

Reply:
Actually that's not needed:

> When a unary fold is used with a pack expansion of length zero, only the following operators are allowed:
>  1) **Logical AND (`&&`)**. The value for the empty pack is `true`
>  2) Logical OR (`||`). The value for the empty pack is `false`
>  3) The comma operator (`,`). The value for the empty pack is `void()`


But it's good to check - I'm adding a test with a single argument to the unit test.